lib/exprengine.cpp

```diff
@@ -997,7 +997,7 @@ ExprEngine::ConditionalValue::Vector ExprEngine::ArrayValue::read(ExprEngine::Va
                 if (i->minValue >= 0 && i->minValue == i->maxValue) {
                     int c = 0;
                     if (i->minValue < stringLiteral->size())
-                        c = stringLiteral->string[i->minValue];
+                        c = stringLiteral->string[static_cast<size_t>(i->minValue)];
```

Comment:
Since this is 128-bit now it will probably cause problems when it doesn't fit into `size_t`.